src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -428,10 +433,11 @@
   </data>
   <data name="MouseWithoutBorders_ToggleEasyMouseShortcut.Header" xml:space="preserve">
     <value>Shortcut to toggle Easy Mouse.</value>
-    <comment>Ctrl and Alt are the keyboard keys</comment>
+    <comment>{Locked="Easy Mouse"}</comment>
   </data>
   <data name="MouseWithoutBorders_ToggleEasyMouseShortcut.Description" xml:space="preserve">
-    <value>Only works if EasyMouse is set to Enabled or Disabled.</value>
+    <value>Only works if Easy Mouse is set to Enabled or Disabled.</value>
```

Comment:
don't lock as it should mimic the setting.  Easy Mouse should be two words too

![image](https://github.com/microsoft/PowerToys/assets/1462282/4465e712-0dec-46f3-ba2f-11255d921b0b)

Reply:
@crutkas I honestly have no idea what you mean here